### ISSUE TITLE
TestCase.php error /w Laravel 5.4.13?

### ISSUE THREAD

**Issue by sandervanhooft** [nothingworksinc/ticketbeast#43]
(Laravel 5.4.13)
When copy-pasting the [TestCase.php](https://github.com/nothingworksinc/ticketbeast/blob/master/tests/TestCase.php) file, I get this error:

```
[Symfony\Component\Debug\Exception\FatalErrorException]                                                                                     
Declaration of class@anonymous::report(Tests\Exception $e) must be compatible with Illuminate\Contracts\Debug\ExceptionHandler::report(Exception $e)    
```

I'm trying to get the `disableExceptionHandling()` method to work. Worked fine on my Laravel 5.3 install.

**Comment by sandervanhooft** [comment]
Nvm, I'll need to repost this issue with better clarification as soon as I have the time.

**Comment by adamwathan** [comment]
Hey @sandervanhooft, add this to the top of your TestCase file:

`use Exception;`

You can see in the error that your TestCase had type-hinted `Tests\Exception` because the TestCase file is namespaced in Laravel 5.4

**Comment by sandervanhooft** [comment]
Hi @adamwathan,

Problem solved! Laravel 5.4.13 features a slightly different TestCase.php skeleton:
```
<?php

namespace Tests;

use Illuminate\Foundation\Testing\TestCase as BaseTestCase;


abstract class TestCase extends BaseTestCase
{
    use CreatesApplication;
}

```

With your suggestion I implemented this (_let me know if there's anything missing_):

```
<?php

namespace Tests;

use App\Exceptions\Handler;
use Exception;
use Illuminate\Contracts\Debug\ExceptionHandler;
use Illuminate\Foundation\Testing\TestCase as BaseTestCase;

abstract class TestCase extends BaseTestCase
{
    use CreatesApplication;

    protected function disableExceptionHandling()
    {
        $this->app->instance(ExceptionHandler::class, new class extends Handler {
            public function __construct() {}
            public function report(Exception $e) {}
            public function render($request, Exception $e) {
                throw $e;
            }
        });
    }
}
```
